backend/lib/endpoints/schema/organizations.js

```diff
@@ -26,22 +26,25 @@ const getOrganizationsSchema = {
 
```

Comment:
Why update the `updateOrganizationSchema` and not the `createOrganizationSchema`?

Reply:
Considering [this comment](https://github.com/FightPandemics/FightPandemics/pull/401#issuecomment-630917180), should I add it in this PR?